devtools/scumm-md5.txt

```diff
@@ -216,7 +216,7 @@ monkey	The Secret of Monkey Island
 	d0b531227a27c6662018d2bd05aac52a	8357	de	DOS	VGA	VGA	4 disk v1.1, 14.Feb.91	Fingolfin
 	66fd5ff9a810dfeb6d6bdada18221140	-1	it	DOS	VGA	VGA	4 disk	Andrea Petrucci
 	45152f7cf2ba8f43cf8a8ea2e740ae09	8357	es	DOS	VGA	VGA	4 disk	Fingolfin
-	a01fab4a64d47b96e2e58e6b0f825cc7	8347	fr	DOS	VGA	VGA	8 disk v1.0, 6/4/91 II	Sebast26
+	a01fab4a64d47b96e2e58e6b0f825cc7	8347	fr	DOS	VGA	VGA	8 disk v1.0, 6/4/91 II (incomplete MT-32)	Sebast26
```

Comment:
scumm-md5.h will also need to be updated with this change

Reply:
@bluegr That's the comment field and the scumm-md5.h file doesn't use it, so rebuilding it would only change the date at the top :)

Reply:
Indeed, thanks!